### ISSUE TITLE
SyntaxError: expected QuotedString

### ISSUE THREAD

**Issue by mwalzer** [PRIDE-Utilities/pride-ontology#16]
That's in [line 2926](https://github.com/PRIDE-Utilities/pride-ontology/blame/master/pride_cv.obo#L2926):
```
def: Studying all protein samples of diverse host environments like the human body, soil or sea water. [PRIDE:PRIDE]
```
Same in 2935 and some

**Comment by ypriverol** [comment]
This is solve?

**Comment by mwalzer** [comment]
fixed with #22
